src/categories/recentreplies.js

```diff
@@ -61,7 +61,8 @@ module.exports = function(Categories) {
 
 	function assignPostsToCategory(category, posts) {
 		category.posts = posts.filter(function(post) {
-			return post.category && parseInt(post.category.cid, 10) === parseInt(category.cid, 10);
+			return !!(post.category && (parseInt(post.category.cid, 10) === parseInt(category.cid, 10)
+			) || (parseInt(post.category.parentCid, 10) === parseInt(category.cid, 10)));
```

Comment:
Yeap, sorry, I was miss clicked there, that's why I closed PR. I'll renew in a few mins with a proper implementation.